Tiltfile

```diff
@@ -26,13 +26,14 @@ settings = {
     "cacppt_version": "v0.4.11",
     "cert_manager_version": "v1.11.0",
     "kustomize_substitutions": {
-        "HIVELOCITY_REGION": "LAX2",
-        "CONTROL_PLANE_MACHINE_COUNT": "3",
-        "WORKER_MACHINE_COUNT": "3",
-        "KUBERNETES_VERSION": "v1.25.2",
-        "HIVELOCITY_IMAGE_NAME": "test-image",
-        "HIVELOCITY_CONTROL_PLANE_MACHINE_TYPE": "hvCustom",
-        "HIVELOCITY_WORKER_MACHINE_TYPE": "hvCustom",
+# use .envrc
```

Comment:
we should either remove this or keep it. Commenting out is no lasting solution

Reply:
@janiskemper yes, you are right. I removed `kustomize_substitutions`.